src/Bridge/RamseyUuid/Serializer/UuidDenormalizer.php

```diff
@@ -21,7 +23,11 @@ final class UuidDenormalizer implements DenormalizerInterface
 {
     public function denormalize($data, $type, $format = null, array $context = [])
     {
-        return Uuid::fromString($data);
```

Comment:
Warning here, these converters will not implement DenormalizerInterface anymore in 3.0 and will have their own interface as the one from the serializer does not fit our needs.

Reply:
Hmm... okay, but does that impact on what I should do here?

Reply:
Oh I thought this was about identifiers conversion mb